### PR TITLE
[opencensus-cpp] Update to 2021-08-26

### PR DESCRIPTION
**Pull request by dllg**
Update to 2021-08-26

- #### What does your PR fix?  
Installs all needed headers and updates to version 62d8281.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  linux / windows, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
Yes

### CODE REVIEW

File: ports/opencensus-cpp/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "opencensus-cpp",
-  "version-string": "0.4.0",
-  "port-version": 2,
+  "version-string": "2021-08-26",
+  "port-version": 3,
```
Comment (JonLiu1993) [inline ports/opencensus-cpp/vcpkg.json:4]:
```suggestion
  "version-date": "2021-08-26",
```

Comment (JonLiu1993) [reply ports/opencensus-cpp/vcpkg.json:4]: @dllg ,Thanks for your pr, if we update port to the new version, we should remove "port-version"

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — COMMENTED** [review]
We now recommand remove "vcpkg_fail_port_install(ON_TARGET "Windows")" in portfile.cmake, since we have used "supports" field in vcpkg.json indicate

**Comment by JonLiu1993** [comment]
@dllg ,Please run command "./vcpkg x-add-version opencensus-cpp --overwrite-version" and commit again

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**